### ISSUE TITLE
[bug]: Audio not playing at Uzbekistan.

### ISSUE THREAD

**Issue by muhammadsiddiqbe** [quran/quran.com-frontend-next#1541]
### Is there an existing issue for this?

- [X] There is no exsiting issue for this bug

### Describe the bug

Assalamu alaikum brother after new update of quran.com audios not playing without VPN. Please check out this issue. Jazakallahu khayran!

![image](https://user-images.githubusercontent.com/62147139/161057450-3bc04ade-7f1e-4ad3-80a4-1baaeaddc987.png)

### Steps to reproduce

1. goto quran.com
2. choose one surah
3. tap the audio button 

### Environment (please complete the following information) and Add any other context about this bug

OS: Windows
Browser: CHROME Version 100.0.4896.60 (Official Build) (64-bit)

### Environment

Production(quran.com)

**Comment by naveed-ahmad** [comment]
- Can you filter the mp3 request( click on media )
- And can you play audio on https://quranicaudio.com/?

**Comment by osamasayed** [comment]
Alsalamu Alikum @muhammadsiddiqbe can you try again? this issue might have been related to our codebase. We just released a deploy that fixes a lot of audio issues. Please try and let me know if the issue still persists.

**Comment by muhammadsiddiqbe** [comment]
![image](https://user-images.githubusercontent.com/62147139/191479046-26897e01-f145-4c90-a683-d8f2ed4ffe52.png)
Sorry guys I missed the notification

> * Can you filter the mp3 request( click on media )
> * And can you play audio on https://quranicaudio.com/?

**Comment by muhammadsiddiqbe** [comment]
![image](https://user-images.githubusercontent.com/62147139/191479247-7fed1c76-abc3-4ab2-bf1a-8cbe8f0c147e.png)
It seems we have an issue with this website while resolving DNS I don't know why it was not working. if I enable VPN it works well. I tried it on many devices and on other networks. Maybe it is blocked by the government

**Comment by muhammadsiddiqbe** [comment]
> Alsalamu Alikum @muhammadsiddiqbe can you try again? this issue might have been related to our codebase. We just released a deploy that fixes a lot of audio issues. Please try and let me know if the issue still persists.

Wa alaikum assalam I think it a DNS problem not regarding codebase

**Comment by muhammadsiddiqbe** [comment]
It was working in the old version of the website.

**Comment by osamasayed** [comment]
> It was working in the old version of the website.

Yes Jazak Allahu Khairan for looking into it. As you mentioned it seems to be a DNS problem. previous.quran.com is using https://audio.qurancdn.com while on quran.com it's using https://download.quranicaudio.com/ which are two different domains and servers. This issue has been reported before by a brother from Uzbakistan as well. We are looking into hosting the audio files somewhere else that is accessible from Uzbakistan. Will let you know inshAllah.

**Comment by muhammadsiddiqbe** [comment]
Any updates brothers? @osamasayed

**Comment by muhammadsiddiqbe** [comment]
@osamasayed Assalamu alaikum brother any news? I think it is not a problem with the IP address of the domain. We should try to change the domain name itself. If it works then we will not have to change to another hosting platform

**Comment by hazem3500** [comment]
@muhammadsiddiqbe seems weird that the CDN would be blocked from Uzbekistan 🤔

I guess relying on a VPN would be a bit annoying 😅 Maybe as a workaround, you can stick with https://previous.quran.com/ until this issue is resolved 🤷 

@osamasayed would it be possible to add `audio.qurancdn.com` as a fallback if `download.quranicaudio.com`? 🤔
I'm not sure if there is a 1:1 mapping between the two CDNs 🤔

**Comment by muhammadsiddiqbe** [comment]
It's been several months since I reported the problem but not been solved yet. I hope you don't forget about it more @hazem3500

**Comment by muhammadsiddiqbe** [comment]
Is there any news about at least what you have planned to fix in the upcoming period brothers?

**Comment by muhammadsiddiqbe** [comment]
Now after more than one year, I am kindly requesting again just a simple thing if you are not going to fix this guys just close the issue.

**Comment by muocod** [comment]
Any news on this? It is been 3 years
